apps/designer/app/shared/shortcuts/index.ts

```diff
@@ -1,14 +1 @@
-export const shortcuts = {
-  esc: "esc",
-  undo: "cmd+z, ctrl+z",
-  redo: "cmd+shift+z, ctrl+shift+z",
-  preview: "cmd+shift+p, ctrl+shift+p",
-  delete: "backspace, delete",
-  breakpointsMenu: "cmd+b, ctrl+b",
-  breakpoint: Array.from(new Array(9))
-    .map((_, index) => `cmd+${index + 1}, ctrl+${index + 1}`)
-    .join(", "),
-  zoom: "cmd+=, ctrl+=, cmd+-, ctrl+-",
-} as const;
-
-export const options = { splitKey: "+", keydown: true };
+export * from "./shortcuts";
```

Comment:
apps/designer/app/shared/shortcuts.ts?

Reply:
I just moved code to a separate file from index, since we use index files only for exporting

Reply:
index files shouldn't contain anything but imports/exports

Reply:
so that we never have to see index files in the editors when working with code

Reply:
Sure, I would just prefer to not use index.ts at all. Or in this case at least do not create folder and index.ts just for one module.

Reply:
Yeah, I don't love it but the parent folder when there is mixture of folders and files doesn't look consistent either, so I thought maybe folder is not that bad and in such cases we also need to use index, because otherwise we have longer paths than necessary.

Reply:
I think overall its a rare case when we use a folder but there is only one file, it should be an edge case

Reply:
it seems unnecessary, but it doesn't really hurt ... so for consistency reasons seems like a better choice